bazel/tools/java/com/code_intelligence/jazzer/tools/FuzzTargetTestWrapper.java

```diff
@@ -185,6 +210,65 @@ private static String rlocationPath(String rootpath) {
     }
   }
 
+  private static void verifyFuzzerOutput(
+      InputStream fuzzerOutput, Set<String> expectedFindings, boolean noHooks) throws IOException {
+    List<String> stackTrace;
+    try (BufferedReader reader = new BufferedReader(new InputStreamReader(fuzzerOutput))) {
+      stackTrace = reader.lines()
+                       .filter(line
+                           -> line.startsWith(EXCEPTION_PREFIX) || line.startsWith(FRAME_PREFIX)
+                               || line.equals(THREAD_DUMP_HEADER))
+                       .collect(toList());
+    }
+    if (expectedFindings.isEmpty()) {
+      if (stackTrace.isEmpty()) {
+        return;
+      }
+      throw new IllegalStateException(String.format(
+          "Did not expect a finding, but got a stack trace:%n%s", String.join("\n", stackTrace)));
+    }
+    if (expectedFindings.contains("thread_dump")) {
+      // Expect THREAD_DUMP_PREFIX as well as at least one frame.
+      if (!stackTrace.contains(THREAD_DUMP_HEADER) || stackTrace.size() < 2) {
+        throw new IllegalStateException(
+            "Expected stack traces for all threads, but did not get any");
+      }
+      if (expectedFindings.size() == 1) {
+        return;
+      }
+    }
+    List<String> findings =
+        stackTrace.stream()
+            .filter(line -> line.startsWith(EXCEPTION_PREFIX))
+            .map(line -> line.substring(EXCEPTION_PREFIX.length()).split(":", 2)[0])
+            .collect(toList());
+    if (findings.isEmpty()) {
+      throw new IllegalStateException("Expected a crash, but did not get a stack trace");
+    }
+    for (String finding : findings) {
+      if (!expectedFindings.contains(finding)) {
```

Comment:
It's not checked if all `expectedFindings` are actually found.

Reply:
This is meant to be a superset of the findings that are allowed rather than the exact set that must be reproduced - different OSes and slightly different OOM semantics make it very difficult to reliably reproduce the exact same set of findings every time.

I would like to clarify the naming and comments though. Do you like `allowedFindings` better?

Reply:
Yes, that clarifies it. Thx.

Reply:
Pushed that as a separate commit to make it a pure refactoring.

Reply:
I like the new name, makes it clear.